src/pages/workspaces/workspace/Data.js

```diff
@@ -425,6 +470,23 @@ const ToolDrawer = withModalDrawer()(({
                 img({ src: dataExplorerLogo, style: { opacity: !dataExplorerButtonEnabled ? .25 : undefined, width: 40 } })
               ]),
               'Data Explorer'
+            ]),
+            h(ModalToolButton, {
+              onClick: () => setToolMode('Notebook'),
+              disabled: !notebookButtonEnabled,
+              tooltip: Utils.cond(
+                [!entityMetadata.cohort, () => 'Talk to your dataset owner about setting up a Data Explorer. See the "Making custom cohorts with Data Explorer" help article.'],
+                [isCohort && entitiesCount > 1, () => 'Select exactly one cohort to open in notebook'],
+                [!isCohort, () => 'Only cohorts can be opened with notebooks'],
+                [notebookButtonEnabled, () => 'Create a Python 2 or 3 notebook with this cohort'],
+                () => undefined
```

Comment:
Unneeded, undefined is the implicit default.

Reply:
Done